mockgcp/mockcloudbuild/workerpool.go

```diff
@@ -77,6 +77,7 @@ func (s *CloudBuildV1) CreateWorkerPool(ctx context.Context, req *pb.CreateWorke
 		result.CreateTime = now
 		result.UpdateTime = now
 		result.State = pb.WorkerPool_RUNNING
+		result.Etag = "W/11111111111111111111111111111111"
```

Comment:
FYI I have some helper functions `computeEtag` that I typically put in utils.go in each mock<service> directory.  (And note that there are two versions, some which return strings, some which return []byte, because some services need string and some []byte)

Would be great to figure out a better place for them, but it's pretty easy to have more realistic etags :-)

Reply:
I like the `computeEtag`!  Make some further changes (with weak indicator), please take a look at the latest commit. If the common `computeEtag` looks good, I can update the mockgcp